### PR TITLE
 fix offer page sidebar

### PR DESCRIPTION
**Pull request by dendoesit**
https://hedvig.atlassian.net/browse/GRW-551

## What?

Removed hasMultipleQuotes conditional as it displayed only when you have 2 quotes.

## Why?

Price on single offer was not displayed.

Currently on dev.hedvigit
image

Now:
image
Multiple quotes 
image

### CODE REVIEW

**Review by styrelius — DISMISSED** [review]
Seems like we suddenly have a `package-lock.json` that shouldn't be here 😅 

Also, I assume you're looking into the type checking that's failing.

**Comment by styrelius** [comment]
@dendoesit Would also be great if you could provide a screenshot of what it looks like with your changes 😊

**Comment by dendoesit** [comment]
hmm. not sure what happened to package.lock . remove and added images with after.

**Review by robinandeer — DISMISSED** [review]
Nice catch! 🧖‍♂️

**Comment by gustaveen** [comment]
Hm, wasn't this what @zakkjellin wanted us to not show, the price when it's only one quote? But I might be wrong. Does this fix the logo issue and cropping issue? @dendoesit

**Comment by dendoesit** [comment]
@gustaveen hmm I wasn't aware of that. I thought it's a bug as the design still displays the text with single offer 😿.  will talk to @zakkjellin  about it.

**Comment by dendoesit** [comment]
@gustaveen  asked Zak
Zak  5:16 PM
Hey Dan! We agreed upon that before, but I'm starting to change my mind about that... Would it be a hassle to "revert" back to showing single contracts too? :thinking_face:

File: src/client/pages/OfferNew/Introduction/Sidebar/index.tsx [hunk 1]
```diff
@@ -231,7 +230,7 @@ export const Sidebar: React.FC<Props> = ({
                 />
               </Header>
               <Body>
-                {hasMultipleQuotes && <PriceBreakdown offerData={offerData} />}
+                {<PriceBreakdown offerData={offerData} />}
```
Comment (guilhermespopolin) [inline src/client/pages/OfferNew/Introduction/Sidebar/index.tsx:233]: I think we can remove the curly brackets now as it should always be displayed :)
```suggestion
                <PriceBreakdown offerData={offerData} />
```

**Review by guilhermespopolin — DISMISSED** [review]

**Review by robinandeer — APPROVED** [review]
Cool! This is the design in Figma also :)

**Pull request merged**